src/Evolution/DgSubcell/Matrices.hpp

```diff
@@ -0,0 +1,228 @@
+ *     </tr>
+ *
+ *     <tr>
+ *     <td> 9 </td> <td> \f$\frac{1663}{1440}\f$ </td>
+ *     <td> \f$\frac{227}{360}\f$ </td>
+ *     <td> \f$\frac{1547}{1152}\f$ </td>
+ *     <td> \f$\frac{245}{288}\f$ </td>
+ *     <td> \f$\frac{3001}{2880}\f$ </td>
+ *     <td> --- </td> <td> --- </td> <td> --- </td>
+ *     </tr>
+ *
+ *     <tr>
+ *     <td> 10 </td> <td> \f$\frac{1375}{1152}\f$ </td>
+ *     <td> \f$\frac{125}{288}\f$ </td>
+ *     <td> \f$\frac{335}{192}\f$ </td>
+ *     <td> \f$\frac{125}{288}\f$ </td>
+ *     <td> \f$\frac{1375}{1152}\f$ </td>
+ *     <td> --- </td> <td> --- </td> <td> --- </td>
+ *     </tr>
+ *
+ *     <tr>
+ *     <td> 11 </td> <td> \f$\frac{1375}{1152}\f$ </td>
+ *     <td> \f$\frac{125}{288}\f$ </td>
+ *     <td> \f$\frac{335}{192}\f$ </td>
+ *     <td> \f$\frac{2483}{5760}\f$ </td>
+ *     <td> \f$\frac{7183}{5760}\f$ </td>
+ *     <td> \f$\frac{863}{960}\f$ </td>
+ *     <td> --- </td> <td> --- </td>
+ *     </tr>
+ *
+ *     <tr>
+ *     <td> 12 </td> <td> \f$\frac{1375}{1152}\f$ </td>
+ *     <td> \f$\frac{125}{288}\f$ </td>
+ *     <td> \f$\frac{335}{192}\f$ </td>
+ *     <td> \f$\frac{2483}{5760}\f$ </td>
+ *     <td> \f$\frac{3583}{2880}\f$ </td>
+ *     <td> \f$\frac{2743}{2880}\f$ </td>
+ *     <td> --- </td> <td> --- </td>
+ *     </tr>
+ *
+ *     <tr>
+ *     <td> 13 </td> <td> \f$\frac{1375}{1152}\f$ </td>
+ *     <td> \f$\frac{125}{288}\f$ </td>
+ *     <td> \f$\frac{335}{192}\f$ </td>
+ *     <td> \f$\frac{2483}{5760}\f$ </td>
+ *     <td> \f$\frac{3583}{2880}\f$ </td>
+ *     <td> \f$\frac{1823}{1920}\f$ </td>
+ *     <td> \f$\frac{2897}{2880}\f$ </td>
+ *     <td> --- </td>
+ *     </tr>
+ *
+ *     <tr>
+ *     <td> 14 </td> <td> \f$\frac{1375}{1152}\f$ </td>
+ *     <td> \f$\frac{125}{288}\f$ </td>
+ *     <td> \f$\frac{335}{192}\f$ </td>
+ *     <td> \f$\frac{2483}{5760}\f$ </td>
+ *     <td> \f$\frac{3583}{2880}\f$ </td>
+ *     <td> \f$\frac{1823}{1920}\f$ </td>
+ *     <td> \f$\frac{5777}{5760}\f$ </td>
+ *     <td> --- </td>
+ *     </tr>
+ *
+ *     <tr>
+ *     <td> 15+ </td> <td> \f$\frac{1375}{1152}\f$ </td>
+ *     <td> \f$\frac{125}{288}\f$ </td>
+ *     <td> \f$\frac{335}{192}\f$ </td>
+ *     <td> \f$\frac{2483}{5760}\f$ </td>
+ *     <td> \f$\frac{3583}{2880}\f$ </td>
+ *     <td> \f$\frac{1823}{1920}\f$ </td>
+ *     <td> \f$\frac{5777}{5760}\f$ </td>
+ *     <td> 1 </td>
+ *     </tr>
+ * </table>
+ *
+ * Note that for brevity we only show half the integration coefficients, the
+ * other half are the same yielding a symmetric stencil about the center of the
+ * cell.
+ */
+template <size_t Dim>
+const DataVector& reconstruction_integration_coefficients(
+    const Mesh<Dim>& dg_mesh, const Index<Dim>& subcell_extents) noexcept;
+
+/*!
+ * \ingroup DgSubcellGroup
+ * \brief Returns the vector of corrections needed to impose the cell average
+ * constraint.
+ */
+template <size_t Dim>
+const DataVector& reconstruction_correction_vector(
```

Comment:
Where is the implementation of this function? Is that coming in a future PR?

Reply:
This and the integration coefficients are no longer needed :)